src/Analyzers/Core/Analyzers/AbstractCodeStyleDiagnosticAnalyzer.cs

```diff
@@ -55,6 +58,7 @@ protected AbstractCodeStyleDiagnosticAnalyzer(ImmutableArray<DiagnosticDescripto
                     DiagnosticSeverity.Hidden,
                     isEnabledByDefault: true,
                     description: description,
+                    helpLinkUri: $"https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/{id}",
```

Comment:
For example, `https://docs.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0011`.